KarhooUISDK/Translations/Base.lproj/Localizable.strings

```diff
@@ -467,6 +463,8 @@
 /* Password Entry Button */
 "Text.Generic.Hide" = "Hide";
 
+"Text.Generic.Locale" = "en-GB";
```

Comment:
What if it's en-US? Or for French fr-CA? Do we support several region codes for the same language?

Reply:
For each new locale we add, we will include the relevant value for the locale. 
e.g. Once we add the strings for fr-CA, it would override the locale value